.github/workflows/testing_and_publishing_OLM_bundle.yml

```diff
@@ -0,0 +1,186 @@
+# action to test our operator lifecycle manager bundle
+# See https://github.com/rabbitmq/OLM-Package-Repo for more info.
+name: test-and-publish-OLM-bundle
+
+on:
+    push:
+      # Pattern matched against refs/tags
+      branches:
+        - "testing_olm"
```

Comment:
main?

Reply:
I will run this one on every release (I left the event on_push on my branch in the meantime in order to better test it)